src/net_processing.cpp

```diff
@@ -339,7 +339,7 @@ bool MarkBlockAsInFlight(NodeId nodeid, const uint256& hash, const CBlockIndex*
     CNodeState *state = State(nodeid);
     assert(state != nullptr);
 
-    // Short-circuit most stuff in case its from the same node
+    // Short-circuit most stuff in case it's from the same node
```

Comment:
Nit: Prefer "it is" to "it's". Applies to "its"/"it's" changes below.

Reply:
Either "it is" or "it's" is acceptable to me.  Since you prefer "it is" I support this.

Reply:
I prefer "it is" since it is unambiguous. "It's" can be "it is" or "it has".

Reply:
Fixed.